### PR TITLE
OSPP 2021-结项报告-基于 Wechaty 开发开源的二师兄社群逗乐机器人

### PR DESCRIPTION
**Pull request by ReiKohaku**
Please make sure to check the following boxes creating your pull request, thank you very much!

- [x] Add Blog Header, including title, author, image, categories, tags, etc.  
- [x] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [x] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [x] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [x] Select a beautiful and meaningful teaser image for your blog
- [x] Create your contributor profile (if you are a first time contributor)

### CODE REVIEW

**Review by Valerieeeee — APPROVED** [review]
LGTM

File: jekyll/_posts/2021-10-06-ospp-final-wechaty-piggy-bro.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  - chat
+  - summer-of-wechaty
+  - summer-2021
+  - ecosystem
+  - ospp
+  - ospp-2021
+  - final
+---
+
+> Author: [@ReiKohaku](https://github.com/ReiKohaku)
```
Comment (lijiarui) [inline jekyll/_posts/2021-10-06-ospp-final-wechaty-piggy-bro.md]: Please remove this part as Mid-Term Demo Day requested.

See: https://docs.google.com/presentation/d/1rWd7aLozgPFyPiSbyxqw1GeGV2-NCGRKvDOUcq9VYcw/edit#slide=id.gec336ad374_1_327

File: jekyll/_posts/2021-10-06-ospp-final-wechaty-piggy-bro.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+* 内置了讲笑话、查天气、搜音乐、看热搜、成语接龙和猜字谜六项基本娱乐功能。
+* 设计了“具备电子感、偏卡通的、类似大众心目中传统二师兄形象”的Logo。
+* 编写了基本的后花园服务和页面，通过阅读文档可以学会部署的方法。后花园提供了使用帮助和状态查询两个模块。
+* 编写了详细的部署文档，以及功能开发相关的文档。
+
+项目原计划的核心目标已经全部完成。
+
+### Live Code视频
+
+{% include iframe.html src="https://www.youtube.com/embed/51EwCNB_Y90" %}
```
Comment (lijiarui) [inline jekyll/_posts/2021-10-06-ospp-final-wechaty-piggy-bro.md:83]: Please add a Chinese version video link:

```
> 国内链接：<国内视频的链接>
```

File: jekyll/_posts/2021-10-06-ospp-final-wechaty-piggy-bro.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+### PPT展示视频
+
+{% include iframe.html src="https://www.youtube.com/embed/Na9pGdf_6oY" %}
```
Comment (lijiarui) [inline jekyll/_posts/2021-10-06-ospp-final-wechaty-piggy-bro.md:89]: Please add a Chinese version video link:

```
> 国内链接：<国内视频的链接>
```

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request.

**Review by lijiarui — APPROVED** [review]
LGTM

**Pull request merged**